packages/legacy/core/App/components/misc/ContentGradient.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { View, StyleSheet } from 'react-native'
+import Svg, { Defs, Rect, LinearGradient, Stop } from 'react-native-svg'
+
+type GradientProps = {
+  backgroundColor: string
+  height?: number
+}
+
+/**
+ * To be used in a relative position controlsContainer that is below (and not in) scrollview content
+ */
+function ContentGradient({ backgroundColor, height = 30 }: GradientProps) {
```

Comment:
Able to use the fat-arrow fn format? Its more in line with what we use elsewhere in the app.

Reply:
Done, good catch